### DISCUSSION TITLE
YAxis `interval` decimal value

### DISCUSSION THREAD

**Discussion by dmdum** [recharts/recharts#4345] [Q&A] [answered]
How do I add an interval with decimal value? I'd like the ticks to be `0.1` apart in values.

**Comment by ckifer** [comment, marked as answer]
You can do something like this [![Edit simple-line-chart (forked)](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/p/sandbox/simple-line-chart-forked-fxlvmh)

It really depends on what your min and max datapoints are. [Interval](https://recharts.org/en-US/api/YAxis#interval) isn't about how far apart things are but rather how many ticks are shown with `0` being all available

**Reply by dmdum** [reply]
Thank you!
